backend/app/api/events.py

```diff
@@ -54,10 +56,78 @@ def query_eventbrite(city: str, date: str, keywords: List[str]) -> List[Event]:
     
     return events
 
+def get_formatted_date(date_entity): 
+    current_date = datetime.now() 
+    # Handle relative dates like "today" or "tomorrow"
+    if date_entity:
+        date_entity = date_entity.lower()
+        if "today" in date_entity:
+            return current_date.replace(hour=0, minute=0, second=0, microsecond=0).isoformat(), current_date.replace(hour=23, minute=59, second=59, microsecond=0).isoformat()
```

Comment:
the today and tomorrow could be basically written as one statement:

```py
if "today" in date_entity or "tomorrow" in date_entity:
   delta = timedelta(days=1) if "tomorrow" else timedelta(days=0)
   start = current_date.replace(hour=0, minute=0, second=0, microsecond=0) + delta 
   end = current_date.replace(hour=23, minute=59, second=59, microsecond=0) + delta 
   return start.isoformat(), end.isoformat()
```

Reply:
or make a function (so we handle all 4 cases)
```py
def get_start_end_date(date):
  start_date = parsed_date.replace(hour=0, minute=0, second=0, microsecond=0)
  end_date = parsed_date.replace(hour=23, minute=59, second=59, microsecond=0)
  return start_date.isoformat(), end_date.isoformat()
```

and then we have:

```py
if "today" in date_entity:
   return get_start_end_date(datetime.now())
elif "tomorrow" in date_entity:
   return get_start_end_date(datetime.now() + timedelta(days=1))
else:
      # Handle YYYY-MM-DD format
      try:
          return get_start_end_date(datetime.strptime(date_entity, "%Y-%m-%d"))
      except ValueError:
          # Handle specific formats like "23-11" (dd-mm)
          try:
             return get_start_end_date(datetime.strptime(date_entity, "%d-%m").replace(year=current_date.year))
          except ValueError:
              return None, None
```

Reply:
fixed